### ISSUE TITLE
Unknown `disksize` config. error

### ISSUE THREAD

**Issue by alexama2012** [boldandbusted/vagrant-kind#5]
Hi @boldandbusted, I followed the quickstart guide in your Readme.md. I cloned the repo like you mentioned  but didn't get any further then `vagrant up` then I receive the following error:
```
$ vagrant up
==> vagrant: A new version of Vagrant is available: 2.2.18 (installed version: 2.2.14)!
==> vagrant: To upgrade visit: https://www.vagrantup.com/downloads.html

Bringing machine 'kind' up with 'virtualbox' provider...
==> kind: Box 'bento/ubuntu-20.04' could not be found. Attempting to find and install...
    kind: Box Provider: virtualbox
    kind: Box Version: >= 0
==> kind: Loading metadata for box 'bento/ubuntu-20.04'
    kind: URL: https://vagrantcloud.com/bento/ubuntu-20.04
==> kind: Adding box 'bento/ubuntu-20.04' (v202107.28.0) for provider: virtualbox
    kind: Downloading: https://vagrantcloud.com/bento/boxes/ubuntu-20.04/versions/202107.28.0/providers/virtualbox.box
Download redirected to host: vagrantcloud-files-production.s3-accelerate.amazonaws.com
==> kind: Successfully added box 'bento/ubuntu-20.04' (v202107.28.0) for 'virtualbox'!
There are errors in the configuration of this machine. Please fix
the following errors and try again:

Vagrant:
* Unknown configuration section 'disksize'.
```
I googled this error and a stackoverflow solution was recommended (https://stackoverflow.com/questions/50063443/vagrant-unknown-configuration-section-disksize). Basically, I need to install a plugin: `vagrant plugin install vagrant-disksize`. Is this a plugin I should have already installed prior to cloning your repo? I'm looking forward to trying vagrant-kind out.

**Comment by alexama2012** [comment]
> Hi @boldandbusted, I followed the quickstart guide in your Readme.md. I cloned the repo like you mentioned but didn't get any further then `vagrant up` then I receive the following error:
> 
> ```
> $ vagrant up
> ==> vagrant: A new version of Vagrant is available: 2.2.18 (installed version: 2.2.14)!
> ==> vagrant: To upgrade visit: https://www.vagrantup.com/downloads.html
> 
> Bringing machine 'kind' up with 'virtualbox' provider...
> ==> kind: Box 'bento/ubuntu-20.04' could not be found. Attempting to find and install...
>     kind: Box Provider: virtualbox
>     kind: Box Version: >= 0
> ==> kind: Loading metadata for box 'bento/ubuntu-20.04'
>     kind: URL: https://vagrantcloud.com/bento/ubuntu-20.04
> ==> kind: Adding box 'bento/ubuntu-20.04' (v202107.28.0) for provider: virtualbox
>     kind: Downloading: https://vagrantcloud.com/bento/boxes/ubuntu-20.04/versions/202107.28.0/providers/virtualbox.box
> Download redirected to host: vagrantcloud-files-production.s3-accelerate.amazonaws.com
> ==> kind: Successfully added box 'bento/ubuntu-20.04' (v202107.28.0) for 'virtualbox'!
> There are errors in the configuration of this machine. Please fix
> the following errors and try again:
> 
> Vagrant:
> * Unknown configuration section 'disksize'.
> ```
> 
> I googled this error and a stackoverflow solution was recommended (https://stackoverflow.com/questions/50063443/vagrant-unknown-configuration-section-disksize). Basically, I need to install a plugin: `vagrant plugin install vagrant-disksize`. Is this a plugin I should have already installed prior to cloning your repo? I'm looking forward to trying vagrant-kind out.


Installing the plugin DID remedy the issue.

**Comment by boldandbusted** [comment]
Yep, that's listed in the 'Prerequisites' section. Should put that in the script in the README. Thanks for the QA. :)
